dbt/macros/schema.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
```

Comment:
This is apparently the recommended way to document macros?

https://docs.getdbt.com/faqs/Docs/documenting-macros